core/services/bulletprooftxmanager/bulletprooftxmanager.go

```diff
@@ -347,6 +347,9 @@ func SendEther(db *gorm.DB, chainID *big.Int, from, to common.Address, value ass
 
 func newAttempt(ethClient eth.Client, ks KeyStore, chainID big.Int, etx EthTx, gasPrice *big.Int, gasLimit uint64) (EthTxAttempt, error) {
 	attempt := EthTxAttempt{}
+	if gasPrice == nil {
+		return attempt, errors.New("gas price missing")
+	}
```

Comment:
This is a minimally intrusive way to reduce this panic to an error.  Could also consider changing the func signature to `gasPrice big.Int`, but it may open a can of worms.

Either way, is this just a secondary problem? Is it valid for `EstimateGas` to return a nil gasPrice without err in the first place?
```
gasPrice, gasLimit, err := eb.estimator.EstimateGas(etx.EncodedPayload, etx.GasLimit)
if err != nil {
	return errors.Wrap(err, "failed to estimate gas")
}
a, err := newAttempt(eb.ethClient, eb.keystore, eb.chainID, *etx, gasPrice, gasLimit)
```

Reply:
It should be impossible for the gasPrice to be nil here. I took a quick look and the only implementation that stands out to me is this one: https://github.com/smartcontractkit/chainlink/blob/e35f15f36bb9a07542fc67251aea17366358df64/core/services/gas/fixed_price_estimator.go#L25

I can't find that snippet on `0.14` releases though, the state in the current master is quite different because of the multi-chain changes.